converter_docs/point_cloud_episodes/lyft.md

```diff
@@ -0,0 +1,414 @@
+<h1 align="left" style="border-bottom: 0"> LYFT Format </h1>
+
+# Overview
+
+Easily import your pointclouds with annotations in the LYFT format. LYFT is an annotation format used in the well-regarded `Lyft Level 5 Prediction` datset. 
```

Comment:
There is no info that data will be uploaded as episodes and that figures of specific object will be matched frame by frame and uploaded as a single object.